test/e2e/framework/test_context.go

```diff
@@ -22,9 +22,8 @@ import (
 	"os"
 	"time"
 
-	"github.com/golang/glog"
-
 	"github.com/kubernetes-sigs/federation-v2/pkg/controller/util"
+	"k8s.io/klog"
```

Comment:
k8s.io is a third party package. It should be better on top of federation package as indepedent section.

Reply:
Agreed. This is a default `gofmt` placement since there's no space.

Reply:
The convention we've converged on is:

```golang
import (
  <system imports (e.g. fmt)>

  <3rd party imports (e.g. glog, errors)>

  <k8s imports (e.g. klog, metav1)>

  <fedv2 imports>
 
  [<test-specific imports>]
)
```